_posts/francois_gerthoffert/2016-10-30-Hellow-World.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Hello World"
+breadcrumb: true
+author: francois_gerthoffert
+date: 2016-10-30
+categories: francois_gerthoffert
+tags:
+    - Blog
+    - Talk
+teaser:
+    info: Welcome to our blog, welcome in our world. We are a team of software engineers, infrastructure specialists and bioinformaticians building infrastructures and tools used by cancer reasearchers around the world. 
+---
+
+Welcome to our blog, welcome in our world. We are a team of software engineers, infrastructure specialists and bioinformaticians building infrastructures and tools used by cancer reasearchers around the world. 
+
+This blog is our opportunity to share with the world how we tackle challenges, find solutions, share our knowledge. All of this applicable not only to cancer research, but to IT in general.
+
+## An idea
+
+Discussions on this topic were triggered by a couple of very interesting sprint demos in which team members showed us solutions and walked us through challenge they faced. 
+
+With very limited documentation available online on those particular topics, it was clear we had knowlegde worth sharing with the world. 
+
+We have a mandate to share and open-source our software, but futher than that, we believe that sharing how we got to that new software version, this cloud infrastructure, or this particular piece of implementation, is also very relevant.
+
+## A technical platform
```

Comment:
Actually, we do use those daily. Not so much on the icgc, but Collaboratory and GDC CMSs are actively being updated those days.